File: test/fixtures/java-rest-controllers/micronaut-user-controller.java
package com.example;

import com.example.dto.UserDto;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

@Controller("/api/users")
public class UserController {
    @Get("/{id}")
    public UserDto get(String id) {
        return null;
    }
}

File: test/fixtures/kotlin-rest-controllers/micronaut-user-controller.kt
package com.example

import com.example.dto.UserDto
import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get

@Controller("/api/users")
class UserController {
    @Get("/{id}")
    fun get(id: String): UserDto {
        throw NotImplementedError()
    }
}
